pip_abandoned/lib.py

```diff
@@ -274,3 +297,16 @@ def search(gh_token, path, verbosity, format_="text"):
     ):
         return 0
     return 9
+
+
+def search_files(gh_token, requirements, verbosity, format_="text"):
+    with TemporaryDirectory() as tempdir:
+        site_packages = create_temp_virtualenv(tempdir)
+
+        for reqs in requirements:
+            subprocess.run(
+                [Path(tempdir) / "bin" / "pip", "install", "-r", reqs.absolute()],
```

Comment:
instead of looping over the files here, pass multiple `-r` arguments to pip and defer the resolution strategy to pip